builder/googlecompute/config.go

```diff
@@ -72,7 +73,7 @@ type Config struct {
 	//     "kmsKeyName": "projects/${project}/locations/${region}/keyRings/computeEngine/cryptoKeys/computeEngine/cryptoKeyVersions/4"
 	//  }
 	//  ```
-	ImageEncryptionKey *compute.CustomerEncryptionKey `mapstructure:"image_encryption_key" required:"false"`
+	ImageEncryptionKey *CustomerEncryptionKey `mapstructure:"image_encryption_key" required:"false"`
```

Comment:
I made `CustomerEncryptionKey` our own type here because `compute.CustomerEncryptionKey` has some other fields that can't be parsed by the generator.